README.md

```diff
@@ -20,6 +20,7 @@ None
 | `x509_certificate_validate_command_secret` | dict of command to validate secret key (see below) | `{"openssl"=>"openssl rsa -check -in %s"}` |
 | `x509_certificate_validate_command_public` | dict of command to validate public key (see below) | `{"openssl"=>"openssl x509 -noout -in %s"}` |
 | `x509_certificate` | keys to manage (see below) | `[]` |
+| `x509_certificate_disable_log` | disable logging of sensitive data during the play if `yes`. note that the log will display the value of `x509_certificate`, including secret key, if `no` | `yes` |
```

Comment:
if it is used for debugging only, `x509_certificate_debug_log` and default to `no` is clear.